recordium/main.py

```diff
@@ -17,16 +17,28 @@
 import logging
 import subprocess
 import sys
+import platform
+import os
+import PyQt5
```

Comment:
This import should be below, between the "from functools..." and the "from PyQt5 ones..." (separated with a blank line from both)